ingestion-beam/src/main/java/com/mozilla/telemetry/io/Read.java

```diff
@@ -52,6 +57,38 @@ public PubsubInput(ValueProvider<String> subscription) {
     }
   }
 
+  /** Implementation of reading from Pub/Sub Lite. */
+  public static class PubsubLiteInput extends Read {
+
+    private final SubscriptionPath path;
+
+    public PubsubLiteInput(ValueProvider<String> subscription) {
+      if (!subscription.isAccessible()) {
+        throw new NotImplementedException(
+            "PubsubLiteIO input requires subscription to be accessible at template creation time.");
+      }
+      path = SubscriptionPath.parse(subscription.get());
+    }
```

Comment:
`PubsubLiteIO` doesn't have any support for `ValueProvider` 😭

Reply:
This may be the push we need to move forward with changing our deployment strategy to Flex Templates: https://github.com/mozilla/gcp-ingestion/issues/1534